package.json

```diff
@@ -28,9 +28,6 @@
   "scripts": {
     "build": "tsc",
     "lint": "eslint . -f visualstudio --report-unused-disable-directives --ext .js,.ts",
-    "run-example": "node build/cli -c ./example/.schemalintrc",
-    "start-example-db": "docker run -d --name dvd-rental -p 54321:5432 kristiandupont/dvdrental-image",
-    "stop-example-db": "docker stop dvd-rental",
```

Comment:
These scripts have been moved to `example/package.json`. If you are using the scripts to run the example against a local version of the library, I would revert this change.